lib/streamlit/elements/widgets/time_widgets.py

```diff
@@ -494,7 +497,7 @@ def _time_input(
     def date_input(
         self,
         label: str,
-        value: DateValue | Literal["today"] = "today",
+        value: DateValue | Literal["today"] = "default_value_today",  # type: ignore[assignment]
```

Comment:
We not add it to type, as Literal["default_value_today"] to not expose it to a user in IDE suggestions